ostelco-ios-client/Models/Region.swift

```diff
@@ -6,6 +6,14 @@
 //  Copyright © 2019 mac. All rights reserved.
 //
 
+func getSimProfileFromSimProfilesArray(_ simProfiles: [SimProfile]) -> SimProfile? {
```

Comment:
Not much logic in there, but copied the way we've implemented fetching a single region from a list of regions in the [Context Model](https://github.com/ostelco/ostelco-ios-client/blob/6297eb39b3f7bd22da1fe18dc1c5d81a4e1300e8/ostelco-ios-client/Models/Context.swift#L11)

Might be more logic to select simProfile later, or maby not

Reply:
Particularly if you're just grabbing the first one, I think this might be premature optimization. You can always add it later!